### PR TITLE
Enlightening Update

### PR DESCRIPTION
**Pull request by Geevies**
* Emergency flares have been replaced by red glowsticks.
* Flares are now slightly less bright and last slightly less longer.
* Glowsticks are now more slightly more bright.
* Glowsticks now only show one message after being activated.
* Flashlights now face the direction you're facing when dropped. They also face a random direction after being thrown.
* Desk lamps are slightly less bright.
* Candles now last twice as long.
* Lanterns are now less bright and only shed light in 120 degrees.
* Diona no longer spawn with flares, as they spawn with the glowstick in their emergency box. And can make their own light.
* You can now hit a light fixture with a broken light again to completely shatter the light bulb.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/devices/lighting/flare.dm [hunk 1]
```diff
@@ -17,9 +17,11 @@
...
 	pickup_sound = 'sound/items/pickup/gloves.ogg'
 
-/obj/item/device/flashlight/flare/New()
-	fuel = rand(800, 1000) // Sorry for changing this so much but I keep under-estimating how long X number of ticks last in seconds.
-	..()
+	var/overrides_activation_message = FALSE
+
+/obj/item/device/flashlight/flare/Initialize()
+	. = ..()
+	fuel = rand(400, 600) // Sorry for changing this so much but I keep under-estimating how long X number of ticks last in seconds.
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/lighting/flare.dm:24]: should remove this comment

File: code/game/objects/items/devices/lighting/flashlight.dm [hunk 2]
```diff
@@ -21,12 +21,19 @@
...
 /obj/item/device/flashlight/Initialize()
 	if (on)
 		light_range = brightness_on
 		update_icon()
 	. = ..()
+	if(light_wedge)
+		var/new_dir = pick(NORTH, SOUTH, EAST, WEST)
+		if(spawn_dir)
+			new_dir = spawn_dir
+		set_dir(new_dir)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/lighting/flashlight.dm:35]: can't you just set_dir(dir) again in initialize without using another var?

**Comment by Chada-1** [comment]
Is it funny that I thought this was related to the new cult changes in the works? 👀

**Pull request merged**